### ISSUE TITLE
Is there a way to return Maybe<Response>

### ISSUE THREAD

**Issue by pmiddend** [alex-shpak/rx-jersey#10]
The title basically says it all. Instead of specifying an entity to return, I want to (also) modify the request to return a redirect, for example. Is that possible at all?

**Comment by pmiddend** [comment]
Okay, then maybe I'm doing it wrong. To illustrate, I created a minimal example, just a few lines of code. The endpoint can be seen [here](https://github.com/pmiddend/rxjersey-maybe-response/blob/master/src/main/java/de/plapadoo/TestEndpoint.java). What's my mistake?

**Comment by alex-shpak** [comment]
Hi!

There is incompatibility right now with Jersey 2.27 since there was big changes in Jersey between 2.26 and 2.27, but it's in WIP.

**Comment by alex-shpak** [comment]
I guess we can close it.

I pushed required changes for Jersey 2.27, unfortunately there are breaking changes
https://github.com/alex-shpak/rx-jersey/tree/feature/jersey-2.27

You can grab it from jitpack: 
```
dependencies {
    implementation 'com.github.alex-shpak:rx-jersey:feature~jersey-2.27-SNAPSHOT'
}```

I will update documentation soon.

**Comment by pmiddend** [comment]
Alright, thanks for the work!

**Comment by pmiddend** [comment]
So you're updating...soon? :laughing:

**Comment by alex-shpak** [comment]
Yeah 😆
Well, actually I was waiting for Dropwizard to release 2.0 with Jersey 2.26+ support, but looks like waiting lasted too long.
https://github.com/dropwizard/dropwizard/milestone/18

Since I use rx-jersey in conjunction with dropwizard I didn't have chance to test 2.27 Jersey in production, unfortunately.
But there is working [example](https://github.com/alex-shpak/rx-jersey/tree/feature/jersey-2.27/example)

If `jitpack` is acceptable option for you, then [here](https://jitpack.io/#alex-shpak/rx-jersey/feature~jersey-2.27-SNAPSHOT)

**Comment by pmiddend** [comment]
We actually just moved away from rx-jersey. Your release policy isn't really compatible with how we're developing (security-wise and all). Still, I'm leaving this open for others to see when things change.

**Comment by alex-shpak** [comment]
Yay! After years of waiting Dropwizard 2.x.x has been released, and now RxJersey 0.12.0 as well, with Jersey 2.30. 🙄
